README.md

```diff
@@ -75,4 +75,18 @@ It's also possible to run individual benchmarks by:
 
 ```
 cargo bench --bench bench ws_round_trip
-```
\ No newline at end of file
+```
+
+## Validate Middleware
+
+This middleware will intercept all method request/responses and compare the result directly with healthy endpoint responses.
+This is useful for debugging to make sure the returned values are as expected.
+You can use this by compiling with `validate` feature and enable validate middleware on your config file.
+```yml
+middlewares:
+  methods:
+    - validate
+```
+NOTE: Keep in mind that if you place `validate` middleware before `inject_params` you may get false positive errors because the request will not be the same.
+
+Use `.validateingore` file to list all the methods you want to ignore.
```

Comment:
can we just configure it in the config.yml file?

Reply:
Yes I did that but it will require to maintain 2 extensions list (one featured) and more code. This implementation is simple

Reply:
but having two config file is just confusing